.github/scripts/pr_comment.js

```diff
@@ -0,0 +1,80 @@
+/*
+Have a single comment on a PR, identified by a comment marker.
+
+Create a new comment if no comment already exists.
+Update the content of the existing comment.
+
+
+https://octokit.github.io/rest.js/v19
+*/
+module.exports = async ({github, context, process}) => {
+    var octokit_rest = github
+    if (context.eventName != "pull_request") {
+        // Only PR are supported.
+        return
+    }
+
+    var sleep = function(second) {
+        return new Promise(resolve => setTimeout(resolve, second * 1000))
+    }
+
+    /*
+    Perform the actual logic.
+
+    This is wrapped so that we can retry on errors.
+    */
+    var doAction = async function() {
+
+        console.log(context)
+
+        fs = require('fs');
+
+        const body = fs.readFileSync(
+            process.env.GITHUB_WORKSPACE + "/" + process.env.COMMENT_BODY, 'utf8');
+        var comment_id = null
+        var comment_marker = '\n' + process.env.COMMENT_MARKER
+        var comment_body =  body + comment_marker
```

Comment:
Looks like a stray space got introduced in https://github.com/twisted/towncrier/pull/447/commits/8208990ecbdbd4454d39a3f461215c7aba748e62#diff-e24afedc0691665628968c647e1b47778e5678bcc154c1c7e83ab9b1d63680b2.

```suggestion
        var comment_body = body + comment_marker
```